deployments/examples/cs3_users_ocis/config/ocis/proxy-config.json

```diff
@@ -65,6 +65,10 @@
           "endpoint": "/app/",
           "backend": "http://localhost:9140"
         },
+        {
+          "endpoint": "/archiver",
```

Comment:
All other `"endpoint"`s have a trailing slash. Is it intended that `/"archiver"` has not?

Reply:
We want `/archiver` because it matches `/archiver` and `/archiver/...` and our actual endpoint is just `/archiver`. `/archiver/` wouldn't match it.

For `/app/` and the other endpoints we could remove it, too. It could make a difference in status codes if somebody calls the path specified in the endpoint (which is not expected in the most cases)